formver.compiler-plugin/uniqueness/src/org/jetbrains/kotlin/formver/uniqueness/ContextTrie.kt

```diff
@@ -3,14 +3,17 @@ package org.jetbrains.kotlin.formver.uniqueness
 import org.jetbrains.kotlin.fir.symbols.FirBasedSymbol
 
 class ContextTrie(
-    val parent: ContextTrie?, val children: MutableMap<FirBasedSymbol<*>, ContextTrie>, override var level: UniqueLevel
+    val parent: ContextTrie?,
+    val symbol: FirBasedSymbol<*>?,
```

Comment:
I think it's a hack to make these optional; I'd rather see an `EmptyContextTrie` object that implements the interface.